app/models/job_invocation_composer.rb

```diff
@@ -98,6 +121,12 @@ def template_with_default
       return template
     end
 
+    private
+
+    def format_datetime(datetime)
+      return datetime if datetime.blank?
+      DateTime.parse(datetime).strftime('%Y-%m-%d %H:%M')
```

Comment:
I think this could raise ArgumentError if user pass invalid date format, which would result in 500 (not tested, can do after rebase)